docs/architecture/adr-070-unordered-account.md

```diff
@@ -2,214 +2,137 @@
 
 ## Changelog
 
-* Dec 4, 2023: Initial Draft (@yihuang, @tac0turtle, @alexanderbez)
-* Jan 30, 2024: Include section on deterministic transaction encoding
+- Dec 4, 2023: Initial Draft (@yihuang, @tac0turtle, @alexanderbez)
+- Jan 30, 2024: Include section on deterministic transaction encoding
+- Mar 12, 2025: Revise implementation to use Cosmos SDK KV Store and require unique timeouts per-address (@technicallyty)
 
 ## Status
 
-ACCEPTED
+Draft
```

Comment:
Update to accpeted whenever we merge

Reply:
So if we're following PROCESS.md, this should probably be `ACCEPTED Not Implemented`